sql/parser/sql.y

```diff
@@ -572,7 +602,10 @@ alter_table_cmd:
   //     [ USING <expression> ]
 | ALTER opt_column name opt_set_data TYPE typename opt_collate_clause alter_using {}
   // ALTER TABLE <name> ADD CONSTRAINT ...
-| ADD table_constraint {}
+| ADD table_constraint
```

Comment:
Err, I misspoke (I confused this with the duplication of `UNIQUE INDEX` and `CONSTRAINT foo UNIQUE` in `CREATE TABLE`). Backing up, the duplication _here_ is weird, but is it worth deviating from postgres over?